ftw/mobilenavigation/browser/resources/navigation.css

```diff
@@ -139,6 +139,15 @@ div.slideNavi.loading {
   min-height: 200px;
   display: block;
 }
+#slider-container {
+  overflow: auto;
+  position: fixed;
+  top:      56px;
```

Comment:
I would use a ``` left: 0; ``` as well for preventing the white border on the left:

![bildschirmfoto 2015-08-24 um 08 39 40](https://cloud.githubusercontent.com/assets/1637820/9434211/b58df20c-4a3c-11e5-92bc-b1d82181733d.png)